src/main/java/sirius/tagliatelle/rendering/GlobalRenderContext.java

```diff
@@ -38,6 +37,10 @@
     protected StringBuilder buffer;
     protected Map<String, String> extraBlocks;
     protected Function<String, String> escaper = GlobalRenderContext::escapeRAW;
+    private static final Pattern OPENING_SCRIPT_TAG = Pattern.compile("<script.*?>", Pattern.CASE_INSENSITIVE);
```

Comment:
Would include a space between the `script` and the `.*` wildcard (also for the style pattern below.
As otherwise <scripting></scripting> (i.e. some custom elements) would increase but not decrease the counters.

Reply:
Good point! It's now implemented.